components/Image/Image.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Image</title>
+    <link rel="stylesheet" href="./Image.css" />
+  </head>
+  <body>
+    <section style="padding: 1rem" class="container flex-centered">
```

Comment:
There is no 'flex-centered' class applied, apart from this everything looks good to me.

Reply:
Hey Kshitiji thanks for review yeah I gave the name wrong I think thanks for pointing it out! Its actually not needed and will be removed later anyways tho